Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp

```diff
@@ -948,17 +948,27 @@ RefPtr<CSSValue> consumeAttr(CSSParserTokenRange args, const CSSParserContext& c
         return nullptr;
 
     CSSParserToken token = args.consumeIncludingWhitespace();
+
     AtomString attrName;
     if (context.isHTMLDocument)
         attrName = token.value().convertToASCIILowercaseAtom();
     else
         attrName = token.value().toAtomString();
 
+    if (!args.atEnd() && !consumeCommaIncludingWhitespace(args))
+        return nullptr;
+
+    RefPtr<CSSValue> fallback = nullptr;
+
+    fallback = consumeDeclarationValue(args, context);
```

Comment:
Ah I figured out what's wrong here, you weren't using the correct method, I was thinking of `CSSParserImpl::consumeDeclarationValue`, not the method in  `CSSPropertyParserHelpers`. You'll also need to pass the `propertyID` argument in `consumeAttr` (the only call site should be using `CSSPropertyContent`, but the spec allows using `attr()` everywhere)

Reply:
@AlexsanderDamaceno Can you use `CSSParser::parseSingleValue` for now? It won't work for shorthands, but should be good enough for your purposes, since `content` is a longhand anyway.